src/installer.js

```diff
@@ -21,15 +39,35 @@ async function installOTP(osVersion, otpVersion) {
  * Install Elixir.
  *
  * @param {string} elixirVersion
+ * @param {string[]} hexMirrors
  */
-async function installElixir(elixirVersion) {
+async function installElixir(elixirVersion, hexMirrors) {
+  if (hexMirrors.length === 0) {
+    throw new Error(
+      `Could not install Elixir ${elixirVersion} from any hex mirror`,
+    )
+  }
+  const [hexMirror, ...hexMirrorsT] = hexMirrors
   const OS = process.platform
-  if (OS === 'linux') {
-    await exec(path.join(__dirname, 'install-elixir.sh'), [elixirVersion])
-  } else if (OS === 'win32') {
-    const script = path.join(__dirname, 'install-elixir.ps1')
-    await exec(`pwsh.exe ${script} -VSN:${elixirVersion}`)
+  try {
+    if (OS === 'linux') {
+      await exec(path.join(__dirname, 'install-elixir.sh'), [
+        elixirVersion,
+        hexMirror,
+      ])
+      return
+    }
+    if (OS === 'win32') {
+      const script = path.join(__dirname, 'install-elixir.ps1')
+      await exec(
+        `pwsh.exe ${script} -VSN:${elixirVersion} -HEX_MIRROR:${hexMirror}`,
+      )
+      return
+    }
+  } catch (err) {
+    core.info(`install-elixir failed for mirror ${hexMirror}`)
```

Comment:
We should output the name of the script (by a variable?) so that's its the actual one that failed.